docs/topics/native/native-definition-file.md

```diff
@@ -0,0 +1,266 @@
+
+```none
+headerFilter = SomeLibrary/**
+```
+
+If `headerFilter` is not provided, all the headers are included. However, we encourage you to use `headerFilter`
+and specify the glob as precisely as possible. In this case, the generated library contains only the necessary
+declarations. It can help avoid various issues when upgrading Kotlin or tools in your development environment.
+
+#### Exclude headers
+
+To exclude specific headers, use the `excludeFilter` property. It can be helpful to remove redundant or problematic
+headers and optimize compilation, as declarations from the specified headers are not included into the bindings:
+
+```none
+excludeFilter = SomeLibrary/time.h
+```
+
+> If the same header is both included with `headerFilter`, and excluded with `excludeFilter`, the specified header will
+> not be included into the bindings.
+>
+{type="note"}
+
+### Import modules
+
+If an Objective-C library has a Clang module, use the `modules` property to specify the module that should be imported:
+
+```none
+modules = UIKit
+```
+
+### Pass compiler and linker options
+
+Use the `compilerOpts` property to pass options to the C compiler, which is used to analyze headers under the hood.
+To pass options to the linker, which is used to link final executables, use `linkerOpts`. For example:
+
+```none
+compilerOpts = -DFOO=bar
+linkerOpts = -lpng
+```
+
+Target-specific options only applicable to the certain target can be specified as well:
+
+```none
+compilerOpts = -DBAR=bar
+compilerOpts.linux_x64 = -DFOO=foo1
+compilerOpts.macos_x64 = -DFOO=foo2
+```
+
+With such a configuration, headers will be analyzed with `-DBAR=bar -DFOO=foo1` on Linux and with `-DBAR=bar -DFOO=foo2`
+on macOS. Note that any definition file option can have both common and platform-specific parts.
+
+### Ignore specific functions
+
+Use the `excludedFunctions` property to specify a list of the function names that should be ignored. This may be
+helpful in case a function declared in the header is not generally guaranteed to be really callable, and
+it's often hard or impossible to figure this out automatically. You can also use this property to work around a bug
+in the interop itself.
+
+### Include a static library
+
+> This feature is [Experimental](components-stability.md#stability-levels-explained). It may be dropped or changed at any time.
+> Use it only for evaluation purposes.
+>
+{type="warning"}
+
+Sometimes it's more convenient to ship a static library with your product, rather than assume it is available within
+the user's environment. To include a static library into `.klib`, use `staticLibrary` and `libraryPaths` properties:
+
+```none
+headers = foo.h
+staticLibraries = libfoo.a
+libraryPaths = /opt/local/lib /usr/local/opt/curl/lib
+```
+
+When given the above snippet the `cinterop` tool will search `libfoo.a` in `/opt/local/lib` and `/usr/local/opt/curl/lib`,
+and if it is found include the library binary into `klib`.
```

Comment:
```suggestion
When given the above snippet, the `cinterop` tool searches `libfoo.a` in `/opt/local/lib` and `/usr/local/opt/curl/lib`,
and if found, includes the library binary in the `klib`.
```
Making it present tense. Should it be `cinterop` or cinterop tool? I see it used in both ways in this doc so can you pick one to use throughout?

Reply:
let's use backticks only when referring to the CL call